scripts/build-testing-sdk.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * This script bundles some scripts used for testing:
+ * - amd loader
+ * - polyfills
+ *   + fetch
+ *   + promises
+ *   + bace64 encoding
+ * - then deps for the testing framework
+ */
+const fs = require('fs');
+
+const sourceModules = getSourceFromModules([
+  // umd module loader
+  'loader.js',
+  // original testing sdk deps
+  'route-recognizer',
+  'fake-xml-http-request',
+  'pretender'
+]);
+
+const outSource = `
+${sourceModules[0]}
+(function (self) {
+;self.fetch = null;
+${sourceModules.slice(1).join('\n')}
+}(window));
+`;
+
+console.log(outSource);
```

Comment:
For flexibility of most of these scripts generated code is sent to stdout.

I could have done something like `process.stdout.write` but I think `console.log` is a bit friendlier.

For this script in particular we do:
`node scripts/build-testing-sdk.js > .dist-test/sdk-testing.js`

And since the script outputs to `stdout` it's very easy in the npm script to change the path in the future or path to another program eg.

`node scripts/build-testing-sdk.js > new-source-folder/new-file-name.js`